product-service/getProductsById.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+const productsMock = require("./productsMock");
+
+module.exports = async (event) => {
+
+  const { productId } = event.pathParameters;
+  const product = productsMock.find((product) => product.id === productId);
+  
+  if (!product) {
+    return {
+      statusCode: 200,
```

Comment:
[404](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404) status code is more appropriate here, or [204](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204)